epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -603,6 +605,36 @@ private MethodSpec generateSetter(ClassToGenerateInfo helperClass, AttributeInfo
         .build();
   }
 
+  private MethodSpec generateReset(ClassToGenerateInfo helperClass) {
+    Builder builder = MethodSpec.methodBuilder("reset")
+        .addModifiers(Modifier.PUBLIC)
+        .returns(VOID);
+
+    for (AttributeInfo attributeInfo : helperClass.getAttributeInfo()) {
+      if (!attributeInfo.hasFinalModifier()) {
+        String attributeName = attributeInfo.getName();
+        TypeName attributeType = attributeInfo.getType();
+        if (attributeType == BOOLEAN) {
+          builder.addStatement("this.$L = false", attributeName);
+        } else if (attributeType == BYTE || attributeType == SHORT || attributeType == INT) {
+          builder.addStatement("this.$L = 0", attributeName);
+        } else if (attributeType == LONG) {
+          builder.addStatement("this.$L = 0L", attributeName);
```

Comment:
Instead of duplicating `builder.addStatement("this.$L = X", attributeName);` we could have each block just state its value
```
String value;
if (attributeType == BOOLEAN) {
   value = "false";
}...
builder.addStatement("this.$L = " + value, attributeName);
```

To clean it up even more there could be a method `getDefaultValue`
```
private String getDefaultValue(TypeName attributeType) {
if (attributeType == BOOLEAN) {
   return "false";
} else ...

return null;
}
```
This could make this a bit easier to read, and reduce the chance of having any typos in all the repeated `"this.$L = 0L"` blocks.